src/os/file_other.go

```diff
@@ -135,3 +135,8 @@ func Readlink(name string) (string, error) {
 func tempDir() string {
 	return "/tmp"
 }
+
+// Truncate is unsupported on this system.
+func (f *File) Truncate(size int64) (err error) {
+	return ErrUnsupported
+}
```

Comment:
Do you want to also add the non-receiver version?

```

// Truncate is unsupported on this system.
func Truncate(string filename, size int64) (err error) {
	return ErrUnsupported
}

func (f *File) Truncate(size int64) (err error) {
	if f.handle == nil {
		return ErrClosed
	}

	return Truncate(f.name, size)
}
```

Reply:
OK change made. Thanks for the suggestion @archie2x